cmd/kubelet/app/options/options.go

```diff
@@ -179,8 +179,8 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 		return fmt.Errorf("invalid node labels: %s", strings.Join(labelErrs, "; "))
 	}
 
-	if f.ContainerRuntime != kubetypes.RemoteContainerRuntime {
-		return fmt.Errorf("unsupported CRI runtime: %q, only %q is currently supported", f.ContainerRuntime, kubetypes.RemoteContainerRuntime)
+	if f.SeccompDefault && !utilfeature.DefaultFeatureGate.Enabled(features.SeccompDefault) {
```

Comment:
Hmm, adding this new code for checking about seccompdefault seems totally unrelated to dropping the container-runtime option